doc/usecases/record_elektra/UC_ansible_export.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: Exporting configuration as Ansible playbook
+
+## Summary
+
+- **Title:** Exporting configuration as Ansible playbook
+- **Scope:** Configuration
+- **Level:** User goal
+- **Actors:** User (usually a sysadmin)
+- **Brief:** User exports certain configuration as Ansible playbook
+
+## Scenarios
+
+- **Precondition:** Some kind of configuration has to exist
```

Comment:
Isn't there some valid "empty" playbook?

Reply:
Yes! we could just create that.